content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -147,7 +147,7 @@ You should set the `.spec.os.name` field to either `windows` or `linux` to indic
 which you want the pod to run. These two are the only operating systems supported for now by 
 Kubernetes. In future, this list may be expanded.
 
-In Kubernetes v{{< skew currentVersion >}}, the value you set for this field has no
+In Kubernetes v{{< skew currentPatchVersion >}}, the value you set for this field has no
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.